march_hardware_builder/robots/test_joint_rotational.yaml

```diff
@@ -1,5 +1,5 @@
 testsetup:
-  ifName: enp2s0
```

Comment:
This should not be changed, the name corresponds with the name on the exoskeleton

Reply:
Well spotted, my mistake :)

Reply:
that was fast.

Reply:
You think I have anything better to do? haha